### PR TITLE
Make display token denomination configurable

### PR DESCRIPTION
**Pull request by webmaster128**
This allows us to use other tokens without forking the codebase.

### CODE REVIEW

File: components/forms/TransactionForm.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ class TransactionForm extends React.Component {
 
     this.state = {
       toAddress: "",
-      amount: 0,
+      amount: "0",
```
Comment (webmaster128) [inline components/forms/TransactionForm.js:29]: With this change, `state.amount` is a string consistently. Before it was initialized as number and was replaced by strings from the field.

File: .env.sample [hunk 2]
```diff
@@ -1,4 +1,6 @@
 FAUNADB_SECRET=
 NEXT_PUBLIC_NODE_ADDRESS=https://cosmoshub.validator.network:443
 NEXT_PUBLIC_DENOM=uatom
-NEXT_PUBLIC_CHAIN_ID=cosmoshub-4
\ No newline at end of file
+NEXT_PUBLIC_DISPLAY_DENOM=ATOM
+NEXT_PUBLIC_DISPLAY_DENOM_EXPONENT=6
```
Comment (samepant) [inline .env.sample:5]: I think you will need to add support for this value in the `printableCoins` function in `lib/displayHelpers.js`, so whenever we show a token amount it's rendered correctly.

Comment (webmaster128) [reply .env.sample:5]: Yeah, good question how to best deal with this. Right now, `printableCoins` magically converts everything starting with `u`, which is good enough for all chains I am aware of. We can remove this automatic conversion and only use this configuration here to convert. But we can also use all 3: the configuration, the automation for `u` and the fallback to the original denom.

**Comment by samepant** [comment]
Just waiting on merge conflicts on this one

**Comment by webmaster128** [comment]
On it!

File: .env.sample (hunk 2 above)
Comment (webmaster128) [reply .env.sample:5]: In the last commit I extended `printableCoin` to also support the configuration and do not rely on auto-conversion only.

**Comment by samepant** [comment]
looks strong! thanks 🥂

**Comment by webmaster128** [comment]
Amazing. That was all I needed to configure the tool for a different network.

I guess you need to set the new options on https://cosmoshub-legacy-multisig.vercel.app/ now to make the deployed version work.

**Comment by samepant** [comment]
> Amazing. That was all I needed to configure the tool for a different network.
> 
> I guess you need to set the new options on https://cosmoshub-legacy-multisig.vercel.app/ now to make the deployed version work.

just did, a new deployment should be live soon

**Pull request merged**